### PR TITLE
Update notification copy

### PR DESCRIPTION
**Pull request by rememberlenny**

### CODE REVIEW

**Comment by rememberlenny** [comment]
This should go green and is finished @jcscottiii @el-mapache @fureigh

File: static_src/actions/user_actions.js [hunk 1]
```diff
@@ -201,19 +201,18 @@ const userActions = {
...
+      description = `The cloud.gov account for ${email} is now associated to this ` +
+        `${entity}. Control their ${entity} roles below.`;
     } else {
-      description = `There was no cloud.gov account found for ${email} ` +
-        'or the user has not verified their account by logging in.' +
-        'They have been sent an email cloud.gov invitation. Their account ' +
-        `has been associated to this ${viewTypeNouns[currentViewedType].singular}` +
-        ` and their ${viewTypeNouns[currentViewedType].singular}` +
-        ' roles can be controlled below.';
+      description = `No verified cloud.gov account was found for ${email}, so ` +
```
Comment (jcscottiii) [inline static_src/actions/user_actions.js]: I'm wondering if the typical user will know what a `verified` account means. I can see just starting off the description with `The user has been sent...`

Comment (rememberlenny) [inline static_src/actions/user_actions.js]: Sounds good!

Comment (rememberlenny) [inline static_src/actions/user_actions.js]: @jcscottiii Updated!

**Pull request merged**